earthpy/spatial.py

```diff
@@ -390,18 +408,21 @@ def plot_rgb(arr, rgb = (0,1,2),
 
     # Then plot. Define ax if it's default to none
     if ax is None:
-      fig, ax = plt.subplots(figsize = figsize)
-    ax.imshow(rgb_bands, extent = extent)
+        fig, ax = plt.subplots(figsize=figsize)
+    else:
+        fig = None
```

Comment:
cool . i haven't set `fig` to `None` before... so it will be interesting to test this out. This is much cleaner as written now as it's more explicit.